packages/x-scheduler/src/joy/event-calendar/EventCalendar.tsx

```diff
@@ -12,9 +12,21 @@ export const EventCalendar = React.forwardRef(function EventCalendar(
   const { events, onEventsChange, className, ...other } = props;
 
   return (
-    <div className="mui light">
-      <div className={clsx(className, 'EventCalendar')} ref={forwardedRef} {...other}>
-        HELLO WORLD ({events.length} events loaded)
+    <div
+      className={clsx(className, 'EventCalendarRoot', 'mui', 'light')}
+      ref={forwardedRef}
+      {...other}
+    >
+      <div className="EventCalendarSidePanel">
```

Comment:
For the class structure I just prefixed everything with `EventCalendar` because all those components are sub-parts of `EventCalendar` and don't make sense outside of its context.

If you prefer `EventCalendar--sidePanel` for example that's fine for me.
Whatever works best for you for the coming weeks of development.